.github/workflows/4_testintegration_inventory-harvester.yml

```diff
@@ -51,4 +51,5 @@ jobs:
         run: |
           cd src
           echo "{}" > states_update_mappings.json
+          echo '{"index_patterns":["wazuh-states-*"],"priority":1,"template":{"settings":{},"mappings":{},"aliases":{}}}' > dummy_template.json
```

Comment:
This overrides all the pre-uploaded indexes.

Reply:
I change the index for this

```json
{"index_patterns":["dummy-*"],"priority":1,"template":{"settings":{},"mappings":{},"aliases":{}}}
```

And the result was correct

```console
================================================================= 70 passed in 1117.64s (0:18:37) =================================================================
```

The idea of use this dummy index is only for the requirement of the module. We can see it on today's meet